frontend/src/components/PrevNext.js

```diff
@@ -1,13 +1,43 @@
-import React, { Component } from 'react';
-
-class PrevNext extends Component {
-  render() {
-    return (
-      <div>
-        
-      </div>
-    );
+import React, {useState, useEffect} from 'react';
+
+function PrevNext() {
+  // fetch local JSON data
+  const [data, setData] = useState([]);
+  const getData=()=>{
+    fetch('data.json',{
+      headers : {
+        'Content-Type': 'application/json',
+        'Accept' : 'application/json'
+      }
+    })
+    .then(function(response){
+      console.log(response)
+      return response.json();
+     })
+     .then(function(myJson){
+       console.log(myJson);
+       setData(myJson)
+     });
   }
```

Comment:
Why are we doing an http request to get a local json file here, can't we read the file instead ?

Reply:
Actually I can